server/backend/mlx.py

```
@@ -54,56 +54,44 @@ def download_model(model_name: str):
+    if isinstance(model_cache_path, str):
+        model_path_str = model_cache_path
+        # Check if we need to load a different model
+        if _current_model_path != model_path_str:
+            # Proactively clean up any previously loaded runner to release memory
+            if _model_cache:
+                try:
+                    for _old_runner in list(_model_cache.values()):
+                        try:
+                            _old_runner.cleanup()
+                        except Exception:
+                            pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)